code/modules/mob/living/carbon/life.dm

```diff
@@ -165,9 +182,12 @@
 		co2overloadtime = 0
 
 	//TOXINS/PLASMA
-	if(Toxins_partialpressure > safe_tox_max)
-		var/ratio = (breath_gases["plasma"][MOLES]/safe_tox_max) * 10
-		if(reagents)
+	if(Toxins_partialpressure > safe_tox_max || Toxins_partialpressure < safe_tox_min)

```

Comment:
This will fuck up with standard plasmaman tox_min/max, which are 16 and 0 respectively